infra/resources/artifacts.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2022 Google LLC
```

Comment:
2023. Fix this in the past PR if we missed it too please.

Reply:
Also update https://github.com/GoogleCloudPlatform/cloud-sql-proxy-operator/blob/main/.github/header-checker-lint.yml to include .tf (and whatever other file types we might need in this repo).

Reply:
Ditto below.

Reply:
Fixed. I updated all the files here to 2023. The infra/permissions are also copyright 2023.